openedx/core/djangoapps/content/block_structure/signals.py

```diff
@@ -16,7 +18,10 @@ def _listen_for_course_publish(sender, course_key, **kwargs):  # pylint: disable
     """
     Catches the signal that a course has been published in the module
     store and creates/updates the corresponding cache entry.
+    Ignores publish signals from content libraries.
     """
+    if isinstance(course_key, LibraryLocator):
+        return
     if config.is_enabled(config.INVALIDATE_CACHE_ON_PUBLISH):
```

Comment:
Nittiest: would an extra line after the `return` statement help with code readability?